### PR TITLE
sci-electronics/librepcb: update older version

### PR DESCRIPTION
**Pull request by Petross404**
Package-Manager: Portage-2.3.51, Repoman-2.3.11

0.1-rc2 version is [out](https://github.com/LibrePCB/LibrePCB/releases/tag/0.1.0-rc2), so I am updating the ebuild.

### CODE REVIEW

**Comment by Petross404** [comment]
The ebuild has errors. I will re-commit ASAP.

**Comment by Petross404** [comment]
Duh, ff didn't show your comment just before I pushed the latest commit. 

Also isn't repoman supposed to add this header as soon as I add the `DCO_SIGNED_OFF_BY="Petros Siligkounas <petross404@gmail.com>" ` in the make.conf?

**Comment by Petross404** [comment]
```
#Fix these variables as soon as a 0.1.0 (no -rcX) release is out.
MY_V="0.1.0"
SRC_URI="https://download.${PN}.org/releases/${MY_V}-rc2/${PN}-${MY_V}-rc2-source.zip"

....

S="${WORKDIR}/${PN}-${MY_V}-rc2"
```
I don't know any other way to use `${PV}` without the `_rcX` and LibrePCB needs 0.1.0-rc2 not 0.1.0_rc2. Please enlighten me :)

**Comment by Petross404** [comment]
Is this PR obsolete based on bug [672054](https://bugs.gentoo.org/672054)?

**Comment by juippis** [comment]
No. If anything, your job got a bit easier ;)
Just update this PR to "bump" to 0.1.0, rebase and force-push. Then remove 0.1.0_px.

What will happen is everyone who had 0.1.0_px installed will have to downgrade (which is not optimal) but hey such is life. 

A hint how you could've solved the _rc2 thing, check how versionator and eapi7-ver works, 
https://devmanual.gentoo.org/eclass-reference/eapi7-ver.eclass/index.html
with EAPI-7 you don't need an extra eclass to inherit them, 
https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html#version-manipulation-and-comparison-commands

**Comment by Petross404** [comment]
> Or you can edit your _rc2 ebuild into 0.1.0, commit, rebase your 3 commits where you **edit** the first commit message and **squash** your 3rd commit into your 1st commit. Then force-push over. I'd say this needs some practice if you're not super familiar with git.

I think I can do this, but bump to newer version and remove older version commits, will be merged in a single one commit. It this ok?

**Comment by juippis** [comment]
>bump to newer version and remove older version commits, will be merged in a single one commit. It this ok?

No, and you can avoid that by rearranging the commits while rebasing. As I said earlier, edit your first, then squash your 3rd into your 1st (you get the idea, however you want to execute it)

**Comment by a17r** [comment]
ping

**Comment by Petross404** [comment]
> 
> 
> ping

I was busy studying for exams the previous month, I will look at this in the following week.

File: sci-electronics/librepcb/librepcb-0.1.0_rc2.ebuild [hunk 1]
```diff
@@ -7,15 +7,16 @@ inherit qmake-utils
 
 DESCRIPTION="Free EDA software to develop printed circuit boards"
 HOMEPAGE="http://librepcb.org/"
```
Comment (a17r) [inline sci-electronics/librepcb/librepcb-0.1.0_rc2.ebuild]: You can use `https` there.

Comment (ktrace) [reply sci-electronics/librepcb/librepcb-0.1.0_rc2.ebuild]: please look https://github.com/gentoo/gentoo/pull/10896, it fresher

**Comment by a17r** [comment]
Superseded by https://github.com/gentoo/gentoo/pull/10896

**Pull request closed without merging**